src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -0,0 +1,363 @@
+        const onSuccess = (file) => {
+            const receipt = file;
+            receipt.state = file && requestType === CONST.IOU.REQUEST_TYPE.MANUAL ? CONST.IOU.RECEIPT_STATE.OPEN : CONST.IOU.RECEIPT_STATE.SCANREADY;
+            setReceiptFile(receipt);
+        };
+
+        IOUUtils.navigateToStartStepIfScanFileCannotBeRead(receiptFilename, receiptPath, onSuccess, requestType, iouType, transactionID, reportID);
+    }, [receiptPath, receiptFilename, requestType, iouType, transactionID, reportID]);
+
+    useEffect(() => {
+        const policyExpenseChat = _.find(participants, (participant) => participant.isPolicyExpenseChat);
+        if (policyExpenseChat) {
+            Policy.openDraftWorkspaceRequest(policyExpenseChat.policyID);
+        }
+    }, [isOffline, participants, transaction.billable, policy]);
+
+    /**
+     * @param {Array} selectedParticipants
+     * @param {String} trimmedComment
+     * @param {File} [receiptObj]
+     */
+    const requestMoney = useCallback(
+        (selectedParticipants, trimmedComment, receiptObj) => {
+            IOU.requestMoney(
+                report,
+                transaction.amount,
+                transaction.currency,
+                transaction.created,
+                transaction.merchant,
+                currentUserPersonalDetails.login,
+                currentUserPersonalDetails.accountID,
+                selectedParticipants[0],
+                trimmedComment,
+                receiptObj,
+                transaction.category,
+                transaction.tag,
+                transaction.billable,
+            );
+        },
+        [report, transaction, currentUserPersonalDetails.login, currentUserPersonalDetails.accountID],
+    );
+
+    /**
+     * @param {Array} selectedParticipants
+     * @param {String} trimmedComment
+     */
+    const createDistanceRequest = useCallback(
+        (selectedParticipants, trimmedComment) => {
+            IOU.createDistanceRequest(
+                report,
+                selectedParticipants[0],
+                trimmedComment,
+                transaction.created,
+                transaction.category,
+                transaction.tag,
+                transaction.amount,
+                transaction.currency,
+                transaction.merchant,
+                transaction.billable,
+                TransactionUtils.getValidWaypoints(transaction.comment.waypoints, true),
+            );
+        },
+        [report, transaction],
+    );
+
+    const createTransaction = useCallback(
+        (selectedParticipants) => {
+            const trimmedComment = lodashGet(transaction, 'comment.comment', '').trim();
+
+            // If we have a receipt let's start the split bill by creating only the action, the transaction, and the group DM if needed
+            if (iouType === CONST.IOU.TYPE.SPLIT && receiptFile) {
+                const existingSplitChatReportID = CONST.REGEX.NUMBER.test(reportID) ? reportID : '';
```

Comment:
We are passing the randomly generated `reportID` as `existingSplitChatReportID` cause a [regression](https://github.com/Expensify/App/issues/34060) more details here https://github.com/Expensify/App/issues/34060#issuecomment-1879274077